public/app/features/explore/Logs/LogsTableWrap.tsx

```diff
@@ -44,14 +45,19 @@ type fieldNameMetaStore = Record<fieldName, fieldNameMeta>;
 
 export function LogsTableWrap(props: Props) {
   const { logsFrames } = props;
+
   // Save the normalized cardinality of each label
   const [columnsWithMeta, setColumnsWithMeta] = useState<fieldNameMetaStore | undefined>(undefined);
 
   // Filtered copy of columnsWithMeta that only includes matching results
   const [filteredColumnsWithMeta, setFilteredColumnsWithMeta] = useState<fieldNameMetaStore | undefined>(undefined);
 
   const height = getTableHeight();
-  const dataFrame = logsFrames[0];
+  // The refId of the current frame being displayed
+  const [currentFrameRef, setCurrentFrameRef] = useState<string | undefined>(logsFrames[0].refId);
+  const [dataFrame, setDataFrame] = useState(
```

Comment:
This is super nit/personal preference, but I would name these consistently and describe them in more detail- e.g. `currentFrameRefId` and `currentFrame` (or `currentDataFrameRefId` and `currentDataFrame`) , cause to me `currentFrameRef` - sounds a little bit like `refObject` and `dataFrame` is quite generic and used at many places. So it would make it more obvious that it is selected frame.

Reply:
Good catch, should be updated now!